doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -839,26 +879,26 @@ KRKNWK-16705: Router device is not fully operational in the distributed network
 
```

Comment:
Please add:

.. rst-class:: v2-5-0

KRKNWK-18572: Bus fault when resetting the Zigbee light switch sample
  Usage of :kconfig:option:`CONFIG_RAM_POWER_DOWN_LIBRARY` leads to a bus fault at reset.

  **Workaround:** Choose  :kconfig:option:`CONFIG_MINIMAL_LIBC` as  :kconfig:option:`CONFIG_LIBC_IMPLEMENTATION`.

Reply:
I believe this would go under Zigbee samples then.

Reply:
Well, it actually affects any usage of `CONFIG_RAM_POWER_DOWN_LIBRARY`.

Reply:
So back under Zigbee protocol?